converter/fluka/helper_parsers/scoring_parser.py

```diff
@@ -30,6 +30,7 @@ class Quantity:
     """Class representing Quantity"""
 
     name: str
+    name_processed = False
```

Comment:
instead of adding `name_processed` I'd add here a field with `output_unit` and proper type (integer ?)
then instead of playing with `name` I'd suggest to modify this logic:
https://github.com/yaptide/converter/blob/4c1a1ed4b6fb55d947c8012abaf62bfa19609464/converter/fluka/cards/scoring_card.py#L83-L103

here the SDUM is written

instead of truncating name to first 10 characters I'd rather use something like:
 - first 4 characters of the name + `_` + some_hash_of_name_and_output_unit_saved_on_5_characters